process/templates/WG_NAME_sandbox_stage.md

```diff
@@ -14,9 +14,13 @@ When contributing to OpenSSF any existing material for the new WG to work on, th
   
 ### Working Group References
 The WG should provide a list of existing resources with links to the repository, and if available, website and any other material to showcase the existing breadth, maturity, and direction of the WG.
-
-| Reference          | URL |
-|--------------------|-----|
-| Repo               |     |
-| Website            |     |
-| Other              |     |
+| Reference           | URL |
+|---------------------|-----|
+| Repo                |     |
+| Meeting Agenda      |     |
+| OSSF Calendar Entry |     |
+| Website             |     |
+| Contributing guide  |     |
+| Security.md         |     |
+| Roadmap             |     |
+| Other               |     |
```

Comment:
```suggestion

| Reference           | URL |
|---------------------|-----|
| Repo                |     |
| Website             |     |
| Security.md    |     |
| Other               |     |
```

According to https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md, once a group has become Sandbox, *then* it sets up regular meetings and gets on the OpenSSF calendar.

I initially didn't think a `Security.md` was part of that, but it looks like it is listed on https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements.